Product/Production/Services/PolicyEngineCore/src/main/java/gov/hhs/fha/nhinc/policyengine/adapter/pip/CdaPdfCreator.java

```diff
@@ -1086,8 +1086,8 @@ private POCDMT000040Component2 createComponent(BinaryDocumentPolicyCriterionType
 
         // Content
         // --------
-        if ((oCriterion.getBinaryDocument() != null) && (oCriterion.getBinaryDocument().length > 0)) {
-            String sBinaryDocument = new String(oCriterion.getBinaryDocument());
+        if (oCriterion.getBinaryDocument() != null && oCriterion.getBinaryDocument().length > 0) {
+            final String sBinaryDocument = new String(oCriterion.getBinaryDocument());
```

Comment:
It is checking the length of the byte array (getBinaryDocument()).  It should be fine to leave it there, as long as it is being done after the null-check.